nautobot/core/choices.py

```diff
@@ -182,14 +182,14 @@ class ButtonActionColorChoices(ChoiceSet):
     Map standard button actions to Bootstrap color classes.
     """
 
-    ADD = "info"
+    ADD = "primary"
     CANCEL = "default"
     CLONE = "success"
     CONFIGURE = "default"
     CONNECT = "success"
     DEFAULT = "default"
     DELETE = "danger"
-    DISCONNECT = "info"
+    DISCONNECT = "danger"
```

Comment:
AFAIK we're not using this constant anywhere in core, but all of the actual "disconnect" buttons (Interface, ConsolePort, etc.) were already hard-coded to `btn-danger` so this is just making it consistent.